launch-or-update-local.ps1

```diff
@@ -24,28 +25,35 @@ param(
         [ValidateSet('true','false')]
         [string]$latest,
         [Parameter()]
-        [switch]$l=$false
+        [switch]$l=$false,
+        [ValidateSet('true','false')]
+        [string]$useRemote,
+        [Parameter()]
+        [switch]$r=$false
     )
 
 $chart="src/buyingcatalogue"
 
-if (($latest -ne "false") -and ($l -eq $false))
+if (($useRemote -ne "false") -and ($r -eq $false))
 {
-    write-host "Getting Latest Chart Versions..."
-    invoke-expression -Command "./update-chart-versions.ps1 -chart $chart -v development"
-}
-else
-{
-    write-host "Getting Master Chart Versions..."
-    invoke-expression -Command "./update-chart-versions.ps1 -chart $chart -v master"
-}
+    if (($latest -ne "false") -and ($l -eq $false))
```

Comment:
Is it right to get the latest chart versions if update is false? won't this cause issues, as the version numbers will be updated, but the dependencies won't be downloaded?

Reply:
I think that switch was mostly for doing master release updates (i.e. just the yaml file). I can remove it if you prefer and leave it as: useRemote? 
yes then always update, 
no then stay as you are. 

That way you're always pulling the latest repos from the acr if you're getting a remote branch

Reply:
So, I think the cases we have are:
- Launch with latest versions from development (download if necessary)
- Launch with latest versions from master (download if necessary)
- Launch with the versions I currently have set up (don't download or update versions)
maybe we should just have a single flag to reflect these scenarios?

Reply:
I've stripped out the update now and left it as master/development (both update) or a second option that stays as you are (no update). Is that ok? (not pushed it yet as I'm troubleshooting a 'hidden return char' bug

Reply:
seems reasonable

Reply:
ok, updated with changes and removed hidden chars from charts.yaml